### ISSUE TITLE
Localization

### ISSUE THREAD

**Issue by hannesj** [HSLdevcom/OpenTripPlanner#12]
We want to have the following in Finnish, Swedish, Sami and English (if available):
- [x] Street names from OSM
- [x] Internal strings on OTP
- [ ] Translations.txt from the various GTFS-feeds

**Comment by hannesj** [comment]
The translation framework is comig from buma/OpenTripPlanner-Maribor@658702de4650960ad3e5720e7b97090fb3029808

**Comment by hannesj** [comment]
The street name localization is currently in buma/OpenTripPlanner-Maribor#1

**Comment by hannesj** [comment]
Translations for GTFS is currently pending on integration of the new GTFS framework to the loader in OTP, so that was moved to a separate issue #20.

**Comment by hannesj** [comment]
Internal strings are localized in opentripplanner/OpenTripPlanner#1762
